packages/shopify-cart/src/types/shopify.type.ts

```diff
@@ -911,32 +1033,32 @@ export type Checkout = Node & {
   shippingDiscountAllocations: Array<DiscountAllocation>;
   /** Once a shipping rate is selected by the customer it is transitioned to a `shipping_line` object. */
   shippingLine?: Maybe<ShippingRate>;
+  /** The price at checkout before shipping and taxes. */
+  subtotalPrice: MoneyV2;
   /**
-   * Price of the checkout before shipping and taxes.
-   * @deprecated Use `subtotalPriceV2` instead.
+   * The price at checkout before duties, shipping, and taxes.
+   * @deprecated Use `subtotalPrice` instead.
    */
-  subtotalPrice: Scalars['Money'];
-  /** The price at checkout before duties, shipping, and taxes. */
   subtotalPriceV2: MoneyV2;
   /** Whether the checkout is tax exempt. */
   taxExempt: Scalars['Boolean'];
   /** Whether taxes are included in the line item and shipping line prices. */
   taxesIncluded: Scalars['Boolean'];
   /** The sum of all the duties applied to the line items in the checkout. */
   totalDuties?: Maybe<MoneyV2>;
+  /** The sum of all the prices of all the items in the checkout, including taxes and duties. */
+  totalPrice: MoneyV2;
   /**
-   * The sum of all the prices of all the items in the checkout, taxes and discounts included.
-   * @deprecated Use `totalPriceV2` instead.
+   * The sum of all the prices of all the items in the checkout, including taxes and duties.
+   * @deprecated Use `totalPrice` instead.
    */
-  totalPrice: Scalars['Money'];
-  /** The sum of all the prices of all the items in the checkout, including duties, taxes, and discounts. */
   totalPriceV2: MoneyV2;
+  /** The sum of all the taxes applied to the line items and shipping lines in the checkout. */
+  totalTax: MoneyV2;
```

Comment:
Do any of these `thingv2` -> `thing` deprecations/changes affect any of our queries?

Reply:
I searched for any uses and couldn't find any. Glad you noticed this too!